pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -0,0 +1,38 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore": "5.0.1-rc3-24126-00",
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc3-24126-00",
+    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24126-00",
```

Comment:
I think we definitely need to think about this some more to think through the consequences. We may need to have a win10 version of CoreCLR or something.

For UWP in particular as you hint at we don't actually pull CoreCLR from the packages as we use the AppX package that contains in which we ship with the CoreRuntimeSDK.